DevModeHandlerTest.start_twoInstances_secondInstanceUsesAnotherPort fails at times

Often one devServer fails the webpack build with:

```
ERROR in Entry module not found: Error: Can't resolve '/tmp/junit7702281413292110741/target/flow-frontend/VaadinDevmodeGizmo.js' in '/tmp/junit7702281413292110741/frontend'

ERROR in ../target/index.ts
Module not found: Error: Can't resolve '@vaadin/flow-frontend/Flow' in '/tmp/junit7702281413292110741/target'
 @ ../target/index.ts 15:0-50 16:33-37
 @ ./generated/vaadin.ts

ERROR in /tmp/junit7702281413292110741/target/index.ts
../target/index.ts
[tsl] ERROR in /tmp/junit7702281413292110741/target/index.ts(18,20)
      TS2307: Cannot find module '@vaadin/flow-frontend/Flow' or its corresponding type declarations.
```

but at times the test fails even when the build passes for both.

--- a/flow-server/src/test/java/com/vaadin/flow/server/DevModeHandlerTest.java
+++ b/flow-server/src/test/java/com/vaadin/flow/server/DevModeHandlerTest.java
@@ -591,14 +591,6 @@ public class DevModeHandlerTest {
         // remove the "singleton" instance to be able to start another one
         removeDevModeHandlerInstance();
 
-        // since the timeout is quite big the server port still should be
-        // available and the second instance should try to reuse it
-
-        DevModeHandler.start(0, createDevModeLookup(), npmFolder,
-                CompletableFuture.completedFuture(null));
-
-        // make checks only if webpack has not yet completed
-
         DevModeHandler anotherHandler = DevModeHandler.start(0,
                 createDevModeLookup(), npmFolder,
                 CompletableFuture.completedFuture(null));

--- a/flow-server/src/test/java/com/vaadin/flow/server/DevModeHandlerTest.java
+++ b/flow-server/src/test/java/com/vaadin/flow/server/DevModeHandlerTest.java
@@ -590,7 +590,9 @@ public class DevModeHandlerTest {
 
         // remove the "singleton" instance to be able to start another one
         removeDevModeHandlerInstance();
-
+        
+        // since the timeout is quite big the server port still should be
+        // available and the second instance should try to reuse it
         DevModeHandler anotherHandler = DevModeHandler.start(0,
                 createDevModeLookup(), npmFolder,
                 CompletableFuture.completedFuture(null));

